src/h5web/visualizations/shared/AxisSystem.tsx

```diff
@@ -19,28 +19,28 @@ function AxisSystem(props: Props): JSX.Element {
   const { camera, size } = useThree();
   const { width, height } = size;
 
-  const { axisDomains, showGrid } = useContext(AxisSystemContext);
+  const { abscissaConfig, ordinateConfig } = useContext(AxisSystemContext);
+  const abscissa = useAbscissaScale();
+  const ordinate = useOrdinateScale();
 
   // axisDomains are the complete domains. visibleDomains change with the camera
-  const [visibleDomains, setVisibleDomains] = useState<AxisDomains>(
-    axisDomains
-  );
-
-  // Finds the visible domains from the camera FOV (zoom and position).
-  const { abscissaScale, abscissaScaleFn } = useAbscissaScale();
-  const { ordinateScale, ordinateScaleFn } = useOrdinateScale();
+  const [visibleDomains, setVisibleDomains] = useState<AxisDomains>({
+    x: abscissa.domain,
+    y: ordinate.domain,
```

Comment:
The hook now returns an object of type `AxisScale`, which includes the domain. The hook already retrieves the domain from the config appropriately, so it seemed smarter to just pass it on than to duplicate the logic here.